spanner/api/Spanner.Samples.Tests/ListInstanceConfigOperationsTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+
+namespace Spanner.Samples.Tests
+{
+    [Collection(nameof(SpannerFixture))]
+    public class ListInstanceConfigOperationsTest
+    {
+        private readonly SpannerFixture _spannerFixture;
+        public ListInstanceConfigOperationsTest(SpannerFixture spannerFixture)
+        {
+            _spannerFixture = spannerFixture;
+        }
+
+        [Fact]
+        public void TestListInstanceConfigOperations()
+        {
+            var sample = new ListInstanceConfigOperationsSample();
+            var configs = sample.ListInstanceConfigOperations(_spannerFixture.ProjectId);
+            Assert.NotEmpty(configs);
```

Comment:
```suggestion
            Assert.NotNull(configs);
```
Same reason as my other comment, we were hitting admin quoata, see https://github.com/GoogleCloudPlatform/dotnet-docs-samples/commit/35baed2aa2a6491523cebfa9d301750088af13b0